Composer/packages/tools/language-servers/language-generation/src/utils.ts

```diff
@@ -132,3 +134,11 @@ export function updateTemplateInContent(content: string, { name, parameters = []
   const resource = LGParser.parse(content);
   return resource.updateTemplate(name, name, parameters, body).toString();
 }
+
+export function loadMemoryVariavles(path: string): object {
```

Comment:
Can you work with @lei9444 and @zhixzhan to figure out a path to make sure we can load from "pre-defined structure" + "the structure which indexer discovered"? 

we may not implement indexer right now in this PR to discover memory paths, but we need make sure the interface is there to use, we can't too static like this

Reply:
Currently, pre-defined structure and user-defined structure will be stored separately. And the completion suggestions will base on both of them. 
pre-defined structure is stored in an object, imported from lib.
user-defined structure will be updated by a trigger event. A fileResolver will handle reading the file contains the content.

Reply:
Let's get sometimes today to discuss this, looks like i was not sure about what the static vs dynamic means.